docker/scripts/build-whisper-tensorrt.sh

```diff
@@ -48,19 +48,21 @@ download_and_build_model() {
     # wget --directory-prefix=assets "$model_url"
     # echo "Download completed: ${model_name}.pt"
     if [ ! -f "assets/${model_name}.pt" ]; then
-        wget --directory-prefix=assets "$model_url"
+        wget --directory-prefix=assets "$model_url" > /dev/null 2>&1
         echo "Download completed: ${model_name}.pt"
     else
         echo "${model_name}.pt already exists in assets directory."
     fi
 
     local output_dir="whisper_${model_name//./_}"
-    echo "$output_dir"
-    echo "Running build script for $model_name with output directory $output_dir"
-    python3 build.py --output_dir "$output_dir" --use_gpt_attention_plugin --use_gemm_plugin --use_bert_attention_plugin --model_name "$model_name"
+    echo "Running TensorRT-LLM build script for $model_name with output directory $output_dir"
+    python3 build.py --output_dir "$output_dir" --use_gpt_attention_plugin --use_gemm_plugin --use_bert_attention_plugin --model_name "$model_name" > /dev/null 2>&1
```

Comment:
I'd propose not to dump the stdout/err to devnull.

This is actually hiding errors that might be useful to know when one is starting the service.

This is especially important when the script continues, even if one of its commands errored. I'd thus also recommend to add `set -e` in the beginning of this and the `run-whisperlive.sh` script.

An example I just ran into: Some cuda error during the trt-build of the engine. After the hidden error, the next echo told me that the engine was built, but it wasn't.